packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -743,7 +743,7 @@ function makeAugmentedSchema(
     let generatedResolvers: any = {
         ...composer.getResolveMethods(),
         ...Object.entries(Scalars).reduce((res, [name, scalar]) => {
-            if (generatedTypeDefs.includes(`scalar ${name}`)) {
+            if (generatedTypeDefs.includes(`scalar ${name}\n`)) {
```

Comment:
This is why the `Date` resolver as added when a `DateTime` was present in the typedefs. `scalar DateTime` includes `scalar Date` :)